### PR TITLE
Add missing settings to config-settings doc

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

File: source/administration/config-settings.rst [hunk 1]
```diff
@@ -3035,6 +3035,17 @@ Path and filename of the license file on disk. On startup, if Mattermost cannot
 | This feature's ``config.json`` setting is ``"LicenseFileLocation": ""`` with string input.  |
 +---------------------------------------------------------------------------------------------+
 
+Go Routine Health Threshold
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Set a threshold on the number of Go Routines when the Mattermost system is considered to be in a healthy state. When Go Routines exceed this limit, a warning is returned in the server logs.
```
Comment (jasonblais) [inline source/administration/config-settings.rst]: I'm not sure if this is accurate, perhaps @hmhealey you can help verify.

File: source/administration/config-settings.rst [hunk 2]
```diff
@@ -3533,6 +3544,17 @@ Enable Tutorial (Experimental)
 | This feature’s ``config.json`` setting is ``"EnableTutorial": true`` with options ``true`` and ``false`` for above settings respectively.  | 
 +--------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable Join Leave Messages in Default Channel (Experimental)
```
Comment (amyblais) [inline source/administration/config-settings.rst]: Is this different from this: https://docs.mattermost.com/administration/config-settings.html?highlight=config%20settings#enable-default-channel-leave-join-system-messages?

**Comment by jasonblais** [comment]
Thanks! Feedback addressed.

**Comment by hmhealey** [comment]
I'm trying to get something written up for RunJobs/RunScheduler, but I need to get some confirmation about how RunScheduler works in an HA environment. I'll comment with that when I get a chance

**Comment by jasonblais** [comment]
Looks great @hmhealey

**Comment by jasonblais** [comment]
@hmhealey added your text to the PR branch. Thanks again 👍

**Pull request merged**